### ISSUE TITLE
Error message "Your Apptentive API key may not be set correctly!" never show up. even with 401 error

### ISSUE THREAD

**Issue by naikling** [apptentive/apptentive-ios#161]
I put the wrong APIKEY with 

instead of
```
[ATConnect sharedConnection].apiKey = AppTent_Id;
```

I put:

```
[ATConnect sharedConnection].apiKey = @"AppTent_Id";
```

So I step through the code in `ATAPIRequest.m`, at line 130 you have the following code:

```
if (sender.statusCode == 401) {
    ATLogDebug(@"Your Apptentive API key may not be set correctly!");
}
```

But the code is never trigger  even with the status code is 401.
It should be easily reproducible if you set the wrong API key and you dont see the message in output box.

**Comment by frankus** [comment]
Thanks. This will be addressed in an upcoming release, by escalating it to an `ATLogError` call.

**Comment by frankus** [comment]
This should be fixed in version 2.0.4.
